atlasdb-service-server/src/main/java/com/palantir/atlasdb/server/AtlasDbServiceServerConfiguration.java

```diff
@@ -25,7 +25,7 @@
 
     private final AtlasDbConfig atlasdb;
 
-    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_ROOT) AtlasDbConfig atlasdb) {
+    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_OBJECT_NAME) AtlasDbConfig atlasdb) {
```

Comment:
This seems super weird. This is usually just a hardcoded string. It seems like AtlasDbConfigs should stay exactly the same while this changes to `@JsonProperty("atlasdb")`.